package.json

```diff
@@ -5,36 +5,40 @@
   "description": "A starter project for AngularJS",
   "repository": "https://github.com/angular/angular-seed",
   "license": "MIT",
+  "dependencies": {
+    "angular": "^1.5.9",
+    "angular-loader": "^1.5.9",
+    "angular-route": "^1.5.9",
+    "html5-boilerplate": "0.0.1"
+  },
   "devDependencies": {
+    "angular-mocks": "^1.5.9",
     "bower": "^1.7.7",
+    "cpx": "^1.5.0",
     "http-server": "^0.9.0",
     "jasmine-core": "^2.4.1",
     "karma": "^0.13.22",
     "karma-chrome-launcher": "^0.2.3",
     "karma-firefox-launcher": "^0.1.7",
     "karma-jasmine": "^0.3.8",
     "karma-junit-reporter": "^0.4.1",
-    "protractor": "^4.0.9"
+    "protractor": "^4.0.9",
+    "shelljs": "^0.7.5"
   },
   "scripts": {
-    "postinstall": "bower install",
-
+    "postinstall": "npm run copy-libs",
     "update-deps": "npm update",
-    "postupdate-deps": "bower update",
-
+    "postupdate-deps": "npm run copy-libs",
```

Comment:
I think `postinstall` only runs when a module is being installed as a dependency, rather than when one is installing the dependencies of the current module.